src/libdrakvuf/os.h

```diff
@@ -202,6 +202,15 @@ typedef struct os_interface
     status_t (*get_pid_from_handle)
     (drakvuf_t drakvuf, drakvuf_trap_info_t* info, addr_t handle, vmi_pid_t* pid);
 
+    status_t (*get_wow_context)
```

Comment:
Please return `bool` instead of `status_t`

Reply:
Btw. what is the key for choosing the return type? I see some functions returning `bool` and some returning `status_t`. It is not clear at the first glance why one or another.

Reply:
If you rebase on the latest master all of them should be returning `bool` ;)